django/test/testcases.py

```diff
@@ -1257,9 +1257,9 @@ class LiveServerThread(threading.Thread):
     Thread for running a live http server while the tests are running.
     """
 
-    def __init__(self, host, static_handler, connections_override=None):
+    def __init__(self, host, port, static_handler, connections_override=None):
```

Comment:
I didn't make it a `0` default kwarg because nothing constructs `LiveServerThread`s except for `LiveServerTestCase._create_server_thread()`, w hich I changed to take the `port` arg into account.

But I guess maintaining backwards compat with any existing non-Django code that manually constructs `LiveServerThread`s is more important than having a prettier method signature. I'll update the PR.